packages/api/src/hooks/useAuthorize.ts

```diff
@@ -14,7 +14,14 @@ const useAuthorize = () => {
 
     const { data, ...rest } = useQuery('authorize', {
         payload: { authorize: current_token || '' },
-        options: { enabled: Boolean(current_token) },
+        options: {
+            enabled: Boolean(current_token),
+            // for authorise request - we cannot affort any race hazards due to it being randomly triggered
+            // e.g. during the process of swithcing account or smth
+            refetchOnWindowFocus: false,
```

Comment:
@wojciech-deriv finally we add this flag to `authorize`, hallelujah! Also would be great to add `staleTime: Infinity`, because every time when we change the account we invalidate `authorize`, but I'm not sure this will be approved right now. But I'm sure we'll get there later, especially when `account_list` endpoint will be added.

Reply:
"But I'm sure we'll get there later, especially when account_list endpoint will be added." -> exactly :)

actually, been thinking, that even without separate endpoint, we could have separate hook with exactly what you mentioned - thinking about that now, shouldn't be as big refactor as I initially thought,